core/src/test/java/org/apache/calcite/adapter/enumerable/EnumUtilsTest.java

```diff
@@ -67,6 +67,17 @@
     assertThat(Expressions.toString(timeStampToLong),
         is("org.apache.calcite.runtime.SqlFunctions.toLongOptional(x)"));
   }
+
+  @Test public void testTypeFromPrimitiveToBox() {
+    final Expression intVariable =
```

Comment:
It would be nice if we can add more test cases to cover all the integer data types conversion.

Reply:
Ok, done.